README.md

```diff
@@ -110,7 +110,7 @@ assert_eq!(playing_errors, vec![]);
 
 ## Features
 
-- For supported commands, see [docs.rs#Token](https://docs.rs/bms-rs/latest/bms_rs/bms/lex/token/enum.Token.html).
+- For supported commands, see [docs.rs#TokenWithPos](https://docs.rs/bms-rs/latest/bms_rs/bms/lex/token/enum.TokenWithPos.html).
```

Comment:
Even now, it's important for users to visit `Token` doc for checking supported commands. So this change makes no sense.